script/upgrades/MU02/MU02-01-Deploy-MedianDeltaBreaker.sol

```diff
@@ -0,0 +1,49 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
```

Comment:
let's remove this otherwise lgtm

Reply:
@ninocomputer Why remove the license identifier?

Reply:
@bayological not the license identifier but the deploy script is not supposed to be here